cinder/tests/unit/volume/drivers/vmware/test_vmware_datastore.py

```diff
@@ -339,9 +343,9 @@ def test_select_datastore(
         get_datastores.assert_called_once_with()
         filter_datastores.assert_called_once_with(
             datastores, size_bytes, profile_id, anti_affinity_ds, aff_ds_types,
-            valid_host_refs=hosts)
+            valid_host_refs=[hosts])
```

Comment:
I would suggest to move the list creation to `hosts = mock.sentinel.hosts` (e.g `hosts = [mock.sentinel.host0, mock.sentinel.host1]`).
Otherwise it reads like you pass a list of hosts (so a nested list).

Reply:
It's a list either way.  I'm not sure what difference this makes.